packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -3,144 +3,171 @@ id: Alert
 section: components
 related: ['Alert', 'Alert group']
 ---
-
-An **alert** is a notification that provides brief information to the user without blocking their workflow.
-
-There are 2 main types of alerts:  
-
-- **Inline alerts**: Inline alerts communicate information about a specific user action on a page. An inline alert appears within the content area and disappears when the user closes it or navigates away from the page.
+ 
+An **alert** is a notification that provides brief information to the user without blocking their workflow.
+ 
+## **Inline alerts**
+### **Bordered inline alerts**
+Bordered inline alerts communicate information about a specific user action on a page. An inline alert appears within the content area and disappears when the user closes it or navigates away from the page.
 
 ![inline notification placement](./img/inline-notification.png)
```

Comment:
I just updated the sketch file so that the artboard on each of these isn't as big (each of these being the more wire-framy images) Do you mind re-exporting and replacing them? Just want to reduce the blank space around all the images!